app/static/js/config.js

```diff
@@ -14,5 +14,5 @@ export default {
     },
   },
   domain: 'http://localhost/',
-  explorer: 'https://blockchain.ki/'
+  explorer: 'https://mintscan.io/'
```

Comment:
You can also set directly `https://mintscan.io/ki-chain/` here, and then use `explorer + 'txs'`, etc